2/src/styles/_companies.scss

```diff
@@ -0,0 +1,84 @@
+.companies {
+  padding-top: 50px;
+  position: relative;
+  margin-bottom: 140px;
+
+  &-item {
+    justify-content: space-around;
+    display: flex !important;
```

Comment:
don't use important

Reply:
I need here important because slick add in HTML display:inline-block and the only way to make it looks good is to use important